src/main/java/se/sundsvall/incident/integration/lifebuoy/LifeBuoyMapper.java

```diff
@@ -11,29 +15,30 @@
 @Component
 @EnableConfigurationProperties(LifeBuoyProperties.class)
 class LifeBuoyMapper {
-
+    
     private final LifeBuoyProperties properties;
-
+    
     LifeBuoyMapper(LifeBuoyProperties lifeBuoyProperties) {
         this.properties = lifeBuoyProperties;
     }
-
-    LifeBuoyRequestWrapper toLifeBuoyyRequest(IncidentDto dto) {
+    
+    LifeBuoyRequestWrapper toLifeBuoyyRequest(IncidentDto dto) throws JsonProcessingException {
+        var objectmapper = new ObjectMapper();
```

Comment:
Då det här ju är en Spring-managerad böna borde det väl funka bra att autowire:a in object-mappern som redan finns?